releases/release-1.31/release-notes/release-notes-draft.json

```diff
@@ -61,19 +59,18 @@
       "feature"
     ],
     "sigs": [
-      "node",
       "api-machinery",
-      "apps"
+      "apps",
+      "node"
     ],
     "feature": true,
     "duplicate": true,
-    "duplicate_kind": true,
-    "is_mapped": true
+    "duplicate_kind": true
   },
   "118148": {
     "commit": "957c9538670b5f7ead2c9ba9ceb9de081d66caa4",
-    "text": "Exposed the kube-scheduler \"/livez\" and \"/readz\" endpoints for health checks that are in compliance with https://kubernetes.io/docs/reference/using-api/health-checks/#api-endpoints-for-health.",
-    "markdown": "Exposed the kube-scheduler \"/livez\" and \"/readz\" endpoints for health checks that are in compliance with https://kubernetes.io/docs/reference/using-api/health-checks/#api-endpoints-for-health. ([#118148](https://github.com/kubernetes/kubernetes/pull/118148), [@linxiulei](https://github.com/linxiulei)) [SIG API Machinery, Scheduling and Testing]",
+    "text": "The kube-scheduler exposes /livez and /readz for health checks that are in compliance with https://kubernetes.io/docs/reference/using-api/health-checks/#api-endpoints-for-health",
+    "markdown": "The kube-scheduler exposes /livez and /readz for health checks that are in compliance with https://kubernetes.io/docs/reference/using-api/health-checks/#api-endpoints-for-health ([#118148](https://github.com/kubernetes/kubernetes/pull/118148), [@linxiulei](https://github.com/linxiulei)) [SIG API Machinery, Scheduling and Testing]",
```

Comment:
This was run with krel v0.17.2 and it looks like the SIGs are removed and missing SIGs are added back in. @cloudmelon